docs/tasks/regionalTSDiagrams.rst

```diff
@@ -0,0 +1,128 @@
+.. _task_oceanRegionalProfiles:
+
+oceanRegionalProfiles
+=====================
+
+An analysis task for plotting T-S (potential temperature vs. salinity)
+diagrams of climatologis in ocean regions.
+
+Component and Tags::
+
+  component: ocean
+  tags: climatology, regions, antarctic, publicObs
+
+Configuration Options
+---------------------
+
+The following configuration options are available for this task::
+
+  [regionalTSDiagrams]
+  ## options related to plotting time series of groups of ocean regions
+
+  # the names of region groups to plot, each with its own section below
+  regionGroups = ['Antarctic Regions']
+
+  # a list of seasons to compute climatologies over
+  seasons = ['ANN']
+
+
+  [TSDiagramsForAntarcticRegions]
+  ## options related to plotting time series of Antarctic regions
+
+  # A geojson file containing Antarctic ocean regions.  Each region must have
+  # 'zmin' and 'zmax' properites in addition to the usual properies for a region
+  # in geometric_features
+  regionMask = 'antarcticRegions.geojson'
+
+  # list of regions to plot or ['all'] for all regions in the masks file.
+  # See "regionNames" in the antarcticRegions masks file in
+  # regionMaskSubdirectory for details.
+  regionNames = ['all']
+
+  # min and max for T and S
+  TBounds = [-2.5, 4]
+  SBounds = [33.8, 34.8]
+
+  # density contour interval
+  rhoInterval = 0.1
+
+  # The color map for depth
+  colormap = cmo.deep_r
+
+  # The minimum and maximum depth over which fields are averaged, default is
+  # to take these values from the geojson feature's zmin and zmax properties.
+  # Add these to a custom config file to override the defaults.
+  # zmin = -1000
+  # zmax = -400
```

Comment:
I assume you mean 'The minimum and maximum depth over which fields are plotted'? (not averaged).
It is great to have the ability to choose a range of depths, as sometimes the T/S diagram gets crowded and specific areas of the plot are not visible because of the high number of points overlapping each other.

Reply:
Thanks!  Both are fixed